cmd/query/main_test.go

```diff
@@ -156,9 +156,9 @@ func TestSliceErrorHTTPResponse(t *testing.T) {
 		},
 		sliceTest{
 			baseTest{
-				name:           "Request which passed all input checks but still should fail",
```

Comment:
Well, you are now changing the meaning of the test, as this one was supposed to test that some Internal Server errors are propagated to 500, and not to something else, but OK, it is probably less important now.